### PR TITLE
Homework4

### PR DESCRIPTION
**Pull request by acejang1994**
New pullrequest

### CODE REVIEW

**Comment by EvanSimpson** [comment]
If you're curious why all of our commits are in there with yours, it's likely because you fetched master while in your branch.

:+1: on the PR though.

**Comment by acejang1994** [comment]
homework4

**Comment by ThomasTheDane** [comment]
It is not a problem that you closed it, but the flow we were going for was to have you guys close the pull request after we have commented and you have a chance to revise if you so desire

File: classes/class04/burgerjoint/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+		}
+	})
+}
+
+index.kitchen = function (req, res) {
+	// list all of the orders in the kitchen
+	
+	Order.find().exec(function(err, results){
+		if (err){
+			res.status(500).send('displaying ingredients did not work');
```
Comment (ThomasTheDane) [inline classes/class04/burgerjoint/routes/index.js:44]: Nice error checking throughout

File: classes/class04/burgerjoint/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+		});
+
+	});	
+}
+
+index.submitOrder = function (req, res) {
+	
+	// add the given list of ingredients to the Order schema
+	
+	console.log(req.body)
```
Comment (ThomasTheDane) [inline classes/class04/burgerjoint/routes/index.js:136]: better to either exclude these from final version or give them a label so when you come back you don't get a bunch of random logs

File: classes/class04/burgerjoint/views/kitchen.handlebars [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>List of Orders</h1>
+
+{{#each in orders}}
```
Comment (ThomasTheDane) [inline classes/class04/burgerjoint/views/kitchen.handlebars:3]: don't need the "in"

File: classes/class04/burgerjoint/views/order.handlebars [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<h1>Order</h1>
+
+<form id="order" action="submitOrder" method="POST">
+	{{#each ingredients}}
+			<p class="ingredient">
+				<input type="checkbox" class="orderForm" id={{this._id}} name={{this.name}}
```
Comment (ThomasTheDane) [inline classes/class04/burgerjoint/views/order.handlebars:6]: don't need all the "this"

File: classes/class04/burgerjoint/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+// var Order = require(path.join(__dirname,'../models/orderModel.js'));
+
+var index = {};
+
+index.home = function(req, res){
+  // res.render('home');
+  res.render("home");
+};
+
+index.listIngredients = function (req, res) {
```
Comment (ThomasTheDane) [inline classes/class04/burgerjoint/routes/index.js:14]: You definitely should have separated things out into multiple routes like "kitchen" and "ingredients." Doing it all in just index is very disorganized

**Pull request closed without merging**